src/smk/tool/geomark/tool-geomark.js

```diff
@@ -122,6 +122,7 @@ include.module( 'tool-geomark', [
 
             this.changedActive( function () {
                 if ( self.active ) {
+                    smk.$viewer.map.pm.removeControls(); // remove Markup toolbar
```

Comment:
Rather than removing and re-adding the toolbar each time (which requires you to ensure config is in sync with other code), maybe take a look at the 'toggleControls' command.